src/ucp/wireup/address.h

```diff
@@ -35,12 +35,13 @@ enum {
 
 
 enum {
-    UCP_ADDRESS_PACK_FLAG_WORKER_UUID = UCS_BIT(0),
-    UCP_ADDRESS_PACK_FLAG_WORKER_NAME = UCS_BIT(1), /* valid only for debug build */
-    UCP_ADDRESS_PACK_FLAG_DEVICE_ADDR = UCS_BIT(2),
-    UCP_ADDRESS_PACK_FLAG_IFACE_ADDR  = UCS_BIT(3),
-    UCP_ADDRESS_PACK_FLAG_EP_ADDR     = UCS_BIT(4),
-    UCP_ADDRESS_PACK_FLAG_TRACE       = UCS_BIT(16) /* show debug prints of pack/unpack */
+    UCP_ADDRESS_PACK_FLAG_WORKER_UUID    = UCS_BIT(0),
+    UCP_ADDRESS_PACK_FLAG_WORKER_NAME    = UCS_BIT(1), /* valid only for debug build */
+    UCP_ADDRESS_PACK_FLAG_DEVICE_ADDR    = UCS_BIT(2),
+    UCP_ADDRESS_PACK_FLAG_IFACE_ADDR     = UCS_BIT(3),
+    UCP_ADDRESS_PACK_FLAG_EP_ADDR        = UCS_BIT(4),
+    UCP_ADDRESS_PACK_FLAG_DISABLE_HW_AMO = UCS_BIT(6),
```

Comment:
why 6? Left 5 for future use?

Reply:
Also, this flag is introduced but does not take any effect. Are you going to update address pack function later in the next PR? maybe introduce this flag there then?
It is hard to understand the need for it without actual implementation.

Reply:
> why 6? Left 5 for future use?

it was a typo, thanks

> maybe introduce this flag there then?

added to this PR